src/v3/src/transformer/main.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { flow } from 'lodash';
```

Comment:
> If importing this way seems cumbersome, you can use [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) to transform named top-level imports like import { throttle, debounce } from 'lodash' into direct import statements.

from same page

Reply:
yeah, i saw that, happy to go with that style as well, but i think we should try to pick one style and be consistent. we can use lint to prevent the style we don't choose

Reply:
I remember we have a jira to analyze how lodash affects the bundle size, even with the per method packages approach, I am not pretty sure how much extra size will lodash add the the bundle (mostly due to the current bundler, webpack4). we can revisit in that jira to decide if still use lodash or not (real single function packages) when we switch to a new version of bundler that can tree shake properly.

Reply:
you're right, our current priorities are not to optimize bundle size so we can take this on properly later.